.htaccess

```diff
@@ -156,14 +156,64 @@
 
 	# Anything in sapphire/en/<version>/contributing is redirected to trunk, the info in there is usually not version specific,
 	# and only updated on master (e.g. contribution and coding guidelines)
-	#RewriteRule ^framework/en/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
-	#RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
-
+	# RewriteRule ^framework/en/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+	# RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+
+	RewriteRule ^changelog/?$ /en/changelogs [R=301,L]
+	RewriteRule ^installation-into-subversion/?$ /en/getting_started/composer [R=301,L]
+	# See https://github.com/sminnee/silverstripe-rssaggregator. Docs no longer exist.
+	# RewriteRule ^modules:rssaggregator/?$ http://addons.silverstripe.org/add-ons/silverstripe/versionfeed [R=301,L]
```

Comment:
Send this to home page of addons site. No need for the above comment. Once the 404 is fixed we are unlikely to revisit.